### PR TITLE
Modified description

### PR DESCRIPTION
**Pull request by andrewdes**
Changed the description as recommended https://github.com/OpenLiberty/guides-common/issues/48

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@
 :page-layout: guide
 :page-duration: 15 minutes
 :page-releasedate: 2017-09-19
-:page-description: Learn how to build and test a simple web application using Maven and Open Liberty
+:page-description: Learn how to build and test a simple web application using a Maven Project Object Model (POM) file and Liberty Maven plugin.
```
Comment (sishida) [inline README.adoc]: "plugin" should be "plug-in."

Add an article ("a" or "the") before "Liberty Maven plug-in."

Comment (andrewdes) [reply README.adoc]: "plugin" is used throughout the entire guide, should I change all those as well?

Comment (sishida) [reply README.adoc]: Yes, the IBM style guide says that "plug-in" should be hyphenated. Sorry I did not catch that earlier!

Comment (andrewdes) [reply README.adoc]: No worries, just changed it all.

Comment (yeekangc) [reply README.adoc]: We need to look into this further.  We used plugins here: https://github.com/WASdev/ci.maven

Comment (sishida) [reply README.adoc]: Here's a link to the IBM style word usage page where I found the info about "plugin" vs. "plug-in": https://apps.na.collabserv.com/wikis/home?lang=en-us#!/wiki/W580a84a67242_4da6_81cf_69e3db4bdbfa/page/Word%20usage The word "plugin" has its own entry. The Merriam Webster online dictionary also seems to prefer "plug-in."

However, "plugin" is OK when it is used as a name and/or formatted as part of the code, as it is with "liberty-maven-plugin" in the link you mentioned (https://github.com/WASdev/ci.maven). It looks like this ci.maven page uses "plug-in" when the word is not used in reference specifically to "liberty-maven-plugin", which is OK. :)

**Pull request closed without merging**